abel/direct.py

```diff
@@ -5,6 +5,10 @@
 from __future__ import unicode_literals
 
 import numpy as np
+if hasattr(np, 'trapezoid'):  # numpy >= 2
```

Comment:
I agree with this code. However, IMO support for python2 should be dropped.

Reply:
This is the *numpy* version, not python version. In this case, v2 is the upcoming version. (But otherwise I agree with you. No need to support Py2 anymore. It, and my beloved print statement without parentheses, are dead.)

Reply:
Opps! I must be getting old.

>  beloved print statement without parentheses

 It tried to make a come back:
 [LWN: The (non-)return of the Python print statement](https://lwn.net/Articles/823292/)

Reply:
OMG! Guido has returned to bring us back our print statement :)

Reply:
Dropping Py2 is planned (#221), but not for right now. Basically, it will allow simplifying the code (and adding type annotations), but I don't think that just declaring that we don't support Py2 anymore without actually doing any improvements is a good thing. :–)

Reply:
Agreed!